Sources/CircularProgress/CircularProgress.swift

```diff
@@ -35,7 +36,7 @@ public final class CircularProgress: NSView {
 		$0.lineWidth = CGFloat(lineWidth)
 	}
 
-	private lazy var cancelButton = with(CustomButton.circularButton(title: "╳", radius: Double(radius), center: bounds.center)) {
+	private lazy var cancelButton: CustomButton = with(.circularButton(title: "╳", radius: Double(radius), center: bounds.center)) {
```

Comment:
Don't do unrelated changes.

Reply:
Scouts honour: Leave Things better than you found them 👔 ...but alright.

Reply:
✅